CharacterCreation/01_03_Species_Tundari.txt

```diff
@@ -119,25 +119,28 @@ subsequent rolls regarding complications or recovery.
 Clawed:
 	Already equipped with built-in weapons, you are practiced at using the large claws
 on your forelimbs to give you an advantage in hand-to-hand combat. 
-[Increased Damage for Hand-to-Hand]
+[Increase damage of Unarmed Strike by {Vaguely balanced number}.]
 
 Soft-footed:
 	An ambush predator, you are built for sneaking up on those around you. You know how to 
 handle your weight and use the knowledge passed down by generations of hunters to keep 
 your presence a secret.
-[Stealth Rolls may be Averaged]
+[You may Roll the Average on any Stealth check.]
 
 Prehensile Tail: 
-	You have spent a long time using your tail to do more than just talk with. You can grab
-onto items, use it to aid in climbing, or even hang from it. 
-[Tail has ½ the carrying capacity of an arm when upright. Full when hanging/dangling]
+	You have spent a long time using your tail to do more than just communicate. You can
+grab onto items, use it to aid in climbing, or even hang from it. 
+[The tail has ½ the carrying capacity of an arm when upright and full carying capacity when
+hanging or dangling]
 	
 	Note: Tails cannot bear the hanging weight of a Large individual.
 
 Toothed: 
 	As a predatory species, you have the customary long teeth that frighten many. 
-To you, they are commonplace, but to many others, they are far from ordinary. 
-[Gain Advantage on initial Intimidation Rolls toward and against Biological beings]
+To you, they are commonplace, but to many others, they induce a primal fear of being
+hunted and eaten. 
+[Gain Advantage on initial Intimidation checks toward and against non-Tundari Biological
+beings.]
```

Comment:
NB: Why wouldn't being toothy work to intimidate other Tundari? Humans are more intimidated by other big, ugly humans.

Reply:
Fair point.